tools/changelog-helper.sh

```diff
@@ -126,10 +126,15 @@ check_or_add_pr() {
         return
     fi
     checked_ids[$id]=1
-    local json=$(gh pr view "${id/#/}" --json title,author)
+    if grep -qF "#$id" <<<"$changelog"; then
```

Comment:
That will find `#300` if the list contains `#3000` through to `#3009`.  A more precise match would be good.

Reply:
Good point. Will probably add `\b` or something around that later.

Reply:
Updated (no \b though).